.buildkite/steps/packer.sh

```diff
@@ -17,66 +17,34 @@ fi
 
 mkdir -p "build/"
 
-# Build a hash of packer files and the agent versions
-packer_files_sha=$(find Makefile "packer/${os}" plugins/ -type f -print0 | xargs -0 sha256sum | awk '{print $1}' | sort | sha256sum | awk '{print $1}')
-internal_files_sha=$(find go.mod go.sum internal/ -type f -print0 | xargs -0 sha256sum | awk '{print $1}' | sort | sha256sum | awk '{print $1}')
```

Comment:
Do we still need to consider changes for `go.mod`, `go.sum`, etc. with `if_changed` for base/stack images?